README.md

```diff
@@ -50,14 +66,4 @@ The following information is visualized:
     - 🔴 Red: if there exist instances waiting late with a critical level
     - 🟠 Orange: if there exist instances waiting late with a risky level
 
-## How to Run
-
-**NOTE**: this tutorial was tested with Node 16 and npm 8.16.0
-
-Run
-```bash
-npm install
-npm start
-```
 
-Open the URL mentioned in the console

```

Comment:
We must keep this, even if it is in the article
This is a common practice to find such commands in the README to easily start the application.